### ISSUE TITLE
Caption translation

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#1070]
## Feature Request

We can also imagine a system where if there's an English or German speaker in the meeting, their speech would be subtitled in French (language of choice in the caption settings). Same thing for the foreign person if they want the captions translated into their language, this could also facilitate exchanges. 🙂

**Comment by lebaudantoine** [comment]
This feature is quite advanced and requires specific models to be deployed, it is not yet on the roadmap, as it requires a huge work backend side.

**Comment by Xavier4492** [comment]
This is a very useful meeting feature. Two details mattered a lot in my tests: each participant should have their own target caption language, and the original transcript should stay available when translation quality drops.

I am building a similar browser meeting flow in World-Meeting, sharing as a reference: https://world-meeting.app/translated-video-meetings
